angular/core/components/notifications/notifications.coffee

```diff
@@ -13,6 +13,7 @@ angular.module('loomioApp').directive 'notifications', ->
       'membership_requested',
       'membership_request_approved',
       'user_added_to_group',
+      'motion_closed',
```

Comment:
Technical debt note: I'd like to move this list out of here and into `notification.rb`, alongside BETA_NOTIFICATIONS